tensorflow/core/api_def/base_api/api_def_SamplingDataset.pbtxt

```diff
@@ -20,5 +20,10 @@ END
 A scalar representing seed2 of random number generator.
 END
   }
-  summary: "Creates a dataset that contains `rate` elements from the `input_dataset`."
+  summary: "Creates a dataset that takes a Bernoulli sample of the contents of another dataset."
+  description: <<END
+There is no direct Python API for creating Datasets of this type. Rewrite rules in 
```

Comment:
Instead of pointing to the C++ source file, mention the Python API to enable this rewrite (`experimental_optimization.filter_with_random_uniform_fusion`).

Reply:
That makes sense. I was actually debating whether to write the docs that way originally. I will replace the ref to the source file with instructions for setting the optimization flag.